ex3/index.js

```diff
@@ -0,0 +1,20 @@
+function ziua_saptamanii(numarzi) {
+    switch (numarzi) {
+      case 1:
+        return "Luni";
+      case 2:
+        return "Marti";
+      case 3:
+        return "Miercuri";
+      case 4:
+        return "Joi";
+      case 5:
+        return "Vineri";
+      case 6:
+        return "Sambata";
+      case 7:
```

Comment:
```suggestion
      case 7, 0:
```
Se poate luat in considerare si 0. 
De exemplu, la Americani, Duminica incepe prima si are index 0